apps/XMRig/install-64

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+version=6.16.2
+
```

Comment:
It appears `install-32` will help you setup xmrig.conf, but install-64 won't?

Reply:
Oh yeah, I've mostly been been working on the `install-32`, so I kinda forgot about `install-64`. I'll add the `config_helper`.